resque.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files  = [ "LICENSE", "README.markdown" ]
   s.rdoc_options      = ["--charset=UTF-8"]
 
-  s.add_dependency "redis-namespace", "~> 1.0.2"
+  s.add_dependency "redis-namespace", "~> 1.3"
```

Comment:
master bumps to `~> 1.6` [here](https://github.com/al2o3cr/resque/pull/2/files#diff-228d8ece821dbe5178f0d9dc16ce52ffR23)